subscriber.go

```diff
@@ -44,7 +45,7 @@ func NewSubscriber(lastEventID string, logger Logger) *Subscriber {
 		EscapedID:           url.QueryEscape(id),
 		RequestLastEventID:  lastEventID,
 		responseLastEventID: make(chan string, 1),
-		out:                 make(chan *Update, 1000),
+		out:                 make(chan *Update, 1000), // update TestSubscriberDoesNotBlockWhenChanIsFull if you change the buffer size
```

Comment:
Maybe this size could be a `const` shared by the tests?

Anyway, LGTM! :) Thanks!

Reply:
Good idea, thank you